README.md

```diff
@@ -30,14 +30,15 @@ This repository will be used for interop tests. Please jump into the issues if y
 
 ```
 # Do the steps in the install section, then
-> cd ..
 > git clone git@github.com:ipfs/js-ipfs.git
 > cd js-ipfs
 > npm install
-> npm link
-> cd ../interop
-> npm link ipfs
-> npm test
+> cd ..
+> npx connect-deps link js-ipfs
+> npx connect-deps connect
```

Comment:
What is the value added by this?

Reply:
i explain here https://github.com/hugomrdias/connect-deps#connect-deps-----

Reply:
@vmx mind sharing your blog post here and making a PR to add it to `connect-deps` readme ?

Reply:
My blog post about this: https://vmx.cx/cgi-bin/blog/index.cgi/when-npm-link-fails%3A2019-08-01%3Aen%2CJavaScript%2Cnpm

Reply:
you need prettier links 😈

Reply:
> you need prettier links smiling_imp

Changing to a proper static website generator has been on my todo list for years (also doing a re-design of the website). This will then end up in nicer links :)